config/routes.rb

```diff
@@ -2,7 +2,8 @@
 
 Rails.application.routes.draw do
   devise_for :users, controllers: {
-    omniauth_callbacks: 'users/omniauth_callbacks'
+    omniauth_callbacks: 'users/omniauth_callbacks',
+    :registrations => "registrations"
```

Comment:
please use only the new hash notation `registrations: 'registrations'` (and single quotes, it's an unfortunate standard we are following)

Reply:
Done.